.env.default

```diff
@@ -1,8 +1,9 @@
 SERVER_HOSTNAME="localhost"
-SERVER_PORT="3000"
+SERVER_PORT="3001"
```

Comment:
Is this change required due to the demo?

Reply:
It's an error. I'm going to put back the original value.